### PR TITLE
org.flameshot.Flameshot

### PR DESCRIPTION
**Pull request by borgmanJeremy**

### CODE REVIEW

**Comment by borgmanJeremy** [comment]
bot, build org.flameshot.Flameshot

**Comment by LeandroStanger** [comment]
bot, build org.flameshot.Flameshot

**Comment by nedrichards** [comment]
Note, the statusnotifier is still an issue given this in my install of the latest testing build:
> failed to register service "org.kde.StatusNotifierItem-3-1"

File: org.flameshot.Flameshot.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+app-id: org.flameshot.Flameshot
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: flameshot
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
```
Comment (barthalion) [inline org.flameshot.Flameshot.yml:9]:
```suggestion
  - --socket=fallback-x11
```

File: org.flameshot.Flameshot.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - --share=network
+  # QtSingleApplication, allow other instances to see log files
+  - --env=TMPDIR=/var/tmp
+  # Allow loading/saving files from anywhere
+  - --filesystem=host
+  # Notification access
+  - --talk-name=org.freedesktop.Notifications
+  # System Tray Icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
```
Comment (barthalion) [inline org.flameshot.Flameshot.yml:23]: This happens to work only if you don't have other flatpaked Qt application running as applications almost always run as PID 2. You'll need `--own-name=org.kde.*` here.

File: org.flameshot.Flameshot.yml (hunk 1 above)
Comment (barthalion) [reply org.flameshot.Flameshot.yml:9]: Actually, I guess that's needed for xwayland applications.

**Comment by borgmanJeremy** [comment]
bot, build org.flameshot.Flameshot

**Comment by borgmanJeremy** [comment]
@barthalion Okay I think this build is ready to go, if you agree I will drop a tag on the upstream repo and update the yaml file here to use the tag.

File: org.flameshot.Flameshot.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+modules:
+  - name: flameshot
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/flameshot-org/flameshot.git
+        #tag: "v0.8.4"
+        bracnh: "master"
```
Comment (barthalion) [inline org.flameshot.Flameshot.yml:33]:
```suggestion
        branch: "master"
```

**Comment by barthalion** [comment]
bot, build org.flameshot.Flameshot

**Pull request closed without merging**